proxy/proxy.go

```diff
@@ -22,9 +22,10 @@ type Proxy struct {
 	client         *docker.Client
 	withDNS        bool
 	dockerBridgeIP string
+	forceIPAM      bool
 }
 
-func NewProxy(targetURL string, withDNS bool) (*Proxy, error) {
+func NewProxy(targetURL string, withDNS, forceIPAM bool) (*Proxy, error) {
```

Comment:
Suspect you meant to use this parameter to initialize the `Proxy` object towards the end of this function.